pkgs/by-name/pa/papers/package.nix

```diff
@@ -0,0 +1,161 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkg-config
+, appstream
+, desktop-file-utils
+, gtk4
+, glib
+, pango
+, atk
+, gdk-pixbuf
+, shared-mime-info
+, itstool
+, gnome
+, poppler
+, ghostscriptX
+, djvulibre
+, libspectre
+, libarchive
+, libsecret
+, wrapGAppsHook
+, librsvg
+, gobject-introspection
+, yelp-tools
+, gspell
+, gsettings-desktop-schemas
+, dbus
+, gst_all_1
+, gi-docgen
+, supportMultimedia ? true # PDF multimedia
+, libgxps
+, supportXPS ? true # Open XML Paper Specification via libgxps
+, withLibsecret ? true
+, libadwaita
+, exempi
+, cargo
+, rustPlatform
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Sorry, it is the first one I package for nixpkgs so I don't know everything. I don't understand the purpose of using "finalAttrs" here rather than "rec" ?

And more, how to pass the attribute "src" to mkDerivation (finalAttrs: { … }) ?

Reply:
Ok I've done the change today :)